### PR TITLE
[new release] omd (2.0.0~alpha1)

### PR DESCRIPTION
**Pull request by nojb**
A Markdown frontend in pure OCaml

- Project page: <a href="https://github.com/ocaml/omd">https://github.com/ocaml/omd</a>

##### CHANGES:

- Lower OCaml requirement to 4.04.2  (ocaml/omd#213, @jfrolich)

- Big refactoring by @nojb. Changes in interface (simplification of code, might
  affect performance a little). To be tested!

### CODE REVIEW

**Comment by nojb** [comment]
The download URL is not correct; it seems to be a problem with `dune-release`...

File: packages/omd/omd.2.0.0~alpha1/opam [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/ocaml/omd.git"
+x-commit-hash: "a01807ec69c03781db9149d8a6759db049c936b8"
+url {
+  src:
+    "https://github.com/ocaml/omd/releases/download/v2.0.0_TILDE_alpha1/omd-v2.0.0~alpha1.tbz"
```
Comment (mseri) [inline packages/omd/omd.2.0.0~alpha1/opam:40]:
```suggestion
    "https://github.com/ocaml/omd/releases/download/2.0.0.alpha1/omd-2.0.0.alpha1.tbz"
```

looks like the released tarball does not have tildes in the name

Comment (mseri) [reply packages/omd/omd.2.0.0~alpha1/opam:40]: I had missed your comment, sorry

Comment (mseri) [reply packages/omd/omd.2.0.0~alpha1/opam]: I merged this in the meantime, so we can check if and where we will need to add upper bounds

Comment (mseri) [reply packages/omd/omd.2.0.0~alpha1/opam]: Mmh the checksums are still wrong...

File: packages/omd/omd.2.0.0~alpha1/opam [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+authors: [
+  "Philippe Wang <philippe.wang@gmail.com>"
+  "Nicolás Ojeda Bär <n.oje.bar@gmail.com>"
+]
+license: "ISC"
+tags: ["org:ocamllabs" "org:mirage"]
+homepage: "https://github.com/ocaml/omd"
+bug-reports: "https://github.com/ocaml/omd/issues"
+depends: [
+  "dune" {>= "2.5"}
```
Comment (mseri) [inline packages/omd/omd.2.0.0~alpha1/opam:20]:
```suggestion
  "ocaml" {>= "4.04"}
  "dune" {>= "2.5"}
```

`String.split_on_char` was introduced in 4.04

**Comment by mseri** [comment]
The ppc64 failure seems to be independent of this.
For the rest I will add the upper bounds omorrow:
```
bound module Omd_representation)
md2mld.0.3.0 (failed: Unbound module Omd_representation)
ocp_reveal.0.1 (failed: Unbound type constructor Omd.element)
sail.0.13 (failed: This variant pattern is expected to have type Omd.block_desc)
sail.0.14 (failed: This variant pattern is expected to have type Omd.block_desc)
vhdlib.0.9.1 (failed: Unable to locate package blktap-dev)
```

**Comment by mseri** [comment]
I am not sure what are the correct licences to pick, so I will not fix the following lint errors:
```
2021-03-09 10:24.26: Error in sail.0.14:            warning 62: License doesn't adhere to the SPDX standard, see https://spdx.org/licenses/: "BSD3"
2021-03-09 10:24.26: Error in sail.0.14:            warning 41: Some packages are mentioned in package scripts or features, but there is no dependency or depopt toward them: "sail"
2021-03-09 10:24.26: Error in sail.0.13:            warning 62: License doesn't adhere to the SPDX standard, see https://spdx.org/licenses/: "BSD3"
2021-03-09 10:24.26: Error in sail.0.13:            warning 41: Some packages are mentioned in package scripts or features, but there is no dependency or depopt toward them: "sail"
2021-03-09 10:24.26: Error in ocp_reveal.0.1:            warning 62: License doesn't adhere to the SPDX standard, see https://spdx.org/licenses/: "LGPL"
```

**Comment by mseri** [comment]
The ppc64 failures seems to be related to a worker issue, I believe they are independent of this PR.

**Comment by nojb** [comment]
Thanks!

**Pull request merged**